File: qlty-cli/tests/lang/csharp/basic.in/FileComplexity.cs
using System;

public class FileComplexity
{
    public static void Main(string[] args)
    {
        int foo = 42;
        
        if (foo > 0)
        {
            if (foo > 10)
            {
                if (foo < 20)
                {
                    if (foo % 2 == 0)
                    {
                        if (foo % 3 == 0)
                        {
                            if (foo % 5 == 0)
                            {
                                if (foo % 7 == 0)
                                {
                                    if (foo % 11 == 0)
                                    {
                                        if (foo % 13 == 0)
                                        {
                                            if (foo % 17 == 0)
                                            {
                                                Console.WriteLine("Nested!");
                                            }
                                        }
                                    }
                                }
                            }
                        }
                    }
                }
            }
        }
    }
}

File: qlty-cli/tests/lang/vbnet/basic.in/FileComplexity.vb
Imports System

Public Class FileComplexity
    Public Shared Sub Main(args As String())
        Dim foo As Integer = 42

        If foo > 0 Then
            If foo > 10 Then
                If foo < 20 Then
                    If foo Mod 2 = 0 Then
                        If foo Mod 3 = 0 Then
                            If foo Mod 5 = 0 Then
                                If foo Mod 7 = 0 Then
                                    If foo Mod 11 = 0 Then
                                        If foo Mod 13 = 0 Then
                                            If foo Mod 17 = 0 Then
                                                Console.WriteLine("Nested!")
                                            End If
                                        End If
                                    End If
                                End If
                            End If
                        End If
                    End If
                End If
            End If
        End If
    End Sub
End Class
